tests/testthat/test-template.R

```diff
@@ -0,0 +1,14 @@
+test_that("generic report knits", {
```

Comment:
Can we add a test for knitting the empty report as well? And maybe bama and nab too, unless there are issues with those?